tests/client/core/api/test_api.js

```diff
@@ -7,18 +7,26 @@ import { ErrorHandler } from 'core/errorHandler';
 import { unexpectedSuccess } from 'tests/client/helpers';
 
 
+const defaultHeaders = {
+  'Content-Type': 'application/json',
+};
+
+function generateHeaders(headerData = defaultHeaders) {
+  return {
+    get: (name) => headerData[name],
```

Comment:
Weird. I noticed usage of `Content-type` elsewhere in tests but figured it was just an alias, like you said.

Hmmm, so what should I do here, just check for `Content-Type` as before? Does `Content-type` work in browsers other than Firefox and Chrome?